neofetch@eson57/neofetch@eson57.nemo_action.in

```diff
@@ -0,0 +1,8 @@
+[Nemo Action]
+_Name=Neofetch
+Exec=<neofetch@eson57/neofetch@eson57.sh %F>
+Icon-Name=dialog-information-symbolic
+Selection=none
+Extensions=any;
+Conditions=dbus org.Cinnamon;desktop;
+Dependencies=gnome-terminal;neofetch;
```

Comment:
Following up on the first comment, you can remove the dependency for the terminal by just retrieving whatever the user has configured as their default. But the Exec line shouldn't have the `%F` parameter since you have `desktop` in the Conditions. (I can modify the validation script to catch this in the future.) This also affects run-speedtest if you can amend that too.

Reply:
Will be done.